app/websockets.py

```
@@ -55,17 +55,27 @@ async def websocket_endpoint(websocket: WebSocket, client_id: str):
 
         # Process messages
         while True:
-            message = await websocket.receive()
+            try:
+                message = await websocket.receive()
+            except RuntimeError as e:
+                if "Cannot call \"receive\" once a disconnect message has been received" in str(e):
+                    logger.info(f"WebSocket for client {client_id} closed by client.")
+                    break
+                else:
+                    raise
```

Pylint (3.3.7)

[refactor] Unnecessary "else" after "break", remove the "else" and de-indent the code inside it

(R1723)